src/cryptsetup/cryptsetup-generator.c

```diff
@@ -49,20 +49,20 @@ static int generate_keydev_mount(const char *name, const char *keydev, char **un
         assert(mount);
 
         r = mkdir_parents("/run/systemd/cryptsetup", 0755);
-        if (r < 0)
+        if (r < 0 && r != -EEXIST)
```

Comment:
this change should be unnecessary: mkdir_parents() (unlike mkdir()) already eats up EEXIST